OpenSim/Common/ModelDisplayHints.h

```diff
@@ -102,6 +102,9 @@ class OSIMCOMMON_API ModelDisplayHints : public Object {
     OpenSim_DECLARE_PROPERTY(show_debug_geometry, bool,
         "Flag to indicate whether or not to show debug geometry, default to false.");
 
+    OpenSim_DECLARE_PROPERTY(decorations_scale_factor, double,
+        "Scale factor to apply to decorations that have no inherent size but should scale with the overall size of the model (e.g. Marker, ContactHalfSpace); defaults to 1.");
```

Comment:
> even if it had a radius you'll have to place it in the xml file. Having multiple numbers that you need to change (one for Markers, another for muscles, a third for Frame etc. doesn't seem to "scale"

More frustrating in my view, than setting the size of markers and Frames individually (which the GUI could help with multi-select edits and other utilities), is the magical scaling of multiple potentially unrelated objects by a mysterious common factor "decorations_scale_factor". I could understand one knob to change ALL decorations (which is what this factor does not do) but to arbitrarily change some odd decorations seems bound to cause confusion.  I could imaging sizing a HalfSpace decoration to appear as a wall or goal, which could be huge, and not also wanting markers on my player's leg to quadruple in size. I just don't see how these geometry should be related. I think decorations should be sized (not scaled) and then a single scale could be applied to collection of different scales :mouse: vs. 🐘